layouts/partials/tw/nav.html

```diff
@@ -1,137 +1,268 @@
-                :data-ga-action="'modal:search:open'"
-                x-data
-                @click.prevent="$dispatch('modal:search:open')"
+          </ul>
+        {{ end }}
+      </li>
+    {{ end }}
+    {{ range site.Menus.main_more }}
+      <li
+        class="
+          mt-4 flex w-full flex-wrap items-center text-sm leading-none sm:absolute sm:-top-6 sm:right-44
+        "
+        :class="toggle ? 'sm:block sm:w-48 sm:rounded-l sm:bg-white sm:overflow-visible sm:pt-2 sm:pb-9 sm:pr-4 sm:pl-8 sm:border-s-5 sm:border-t sm:border-l sm:border-b md:-top-8 md:pt-[15px] box-content' : 'sm:overflow-hidden sm:w-0'"
+      >
+        <a
+          class="
+            sm:h block w-full flex-1 border-b py-4 font-bold uppercase hover:underline hover:underline-offset-2 sm:flex sm:w-full sm:items-center sm:capitalize
+          "
+          :class="toggle ? 'sm:py-0 sm:h-11 md:justify-start' : 'md:py-0 md:justify-end'"
+          href="{{ .URL }}"
+          data-ga-category="item"
+          >{{ .Name }}</a
+        >
+        {{ if .HasChildren }}
+          <ul class="my-3 flex w-full flex-wrap gap-2">
+            {{ range .Children }}
+              <li
+                class="flex h-6 flex-[1_0_45%] items-center text-sm leading-none sm:flex-[0_0_100%]"
               >
-                Search
-              </a>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
+                <a
+                  class="flex-1 hover:underline hover:underline-offset-2"
+                  href="{{ .URL }}"
+                  data-ga-category="item"
+                  >{{ .Name }}</a
+                >
+              </li>
+            {{ end }}
+          </ul>
+        {{ end }}
+      </li>
+    {{ end }}
+  </ul>
 </nav>
-{{ partial "js/view-trigger.html" "nav" }}
+<span
+  class="
+  opacity-1 mx-0 box-content
+  box-content box-content flex
+  h-auto w-0
+  w-0 w-auto translate-y-1.5
+  rotate-45 overflow-hidden rounded border-s-5 px-3 px-4 px-5 pb-10 pt-12 opacity-0 max-sm:absolute max-sm:border sm:-right-16 sm:-right-16 sm:-top-6 sm:top-52
+  sm:-ml-3 sm:mr-3 sm:block sm:block sm:block sm:h-11 sm:h-11 sm:h-40 sm:w-0 sm:w-0 sm:w-0 sm:w-48 sm:w-48 sm:w-48 sm:w-48 sm:overflow-hidden sm:overflow-hidden sm:overflow-hidden sm:overflow-visible sm:overflow-visible sm:overflow-visible sm:overflow-visible sm:rounded-l
+  sm:rounded-br sm:rounded-tr sm:border-b sm:border-b sm:border-l
+  sm:border-r sm:border-r sm:border-t sm:border-t sm:border-s-5
+  sm:border-s-5 sm:border-s-5 sm:bg-white sm:bg-white sm:bg-white sm:px-16 sm:px-3 sm:py-0 sm:py-0 sm:pb-0 sm:pb-10 sm:pb-3 sm:pb-[35px] sm:pl-4 sm:pl-4 sm:pl-8 sm:pr-4 sm:pr-8
+  sm:pr-8 sm:pt-0 sm:pt-2 sm:pt-2 sm:pt-6 md:absolute md:-top-4 md:-top-8 md:right-1 md:top-0 md:hidden md:h-auto md:w-0 md:w-full md:justify-start md:justify-start md:justify-end md:justify-end
+  md:overflow-visible md:py-0 md:py-0 md:pt-[15px] md:pt-[15px]
+"
```

Comment:
Add a `<!-- comment -->` explaining why we need the invisible spans for CSS purge.

Reply:
https://github.com/spotlightpa/poor-richard/pull/422/commits/9e0d40451ab400ba67268e0e18907b7a28bc9df6